app/helpers/forms/flow_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Forms::FlowHelper
+  def after_login_next_step
+    if Services::Feature.trn_required? && query_store.current_user.trn.blank?
```

Comment:
To allow TRN-free users now all we'll need to do is toggle Services::Feature.trn_required? so that it returns false and users without TRNs will continue on with the flow as any other user would, eventually reaching lead providers. For now this is hardcoded to true and all users without TRNs go to the qualified_teacher_check question.